mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java

```diff
@@ -236,11 +236,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         processIntent(getIntent());
 
-        if (isFullscreenEnabled()) {
+        if (Util.isFullscreenEnabled(this)) {
             IntentFilter filter = new IntentFilter(Intent.ACTION_DREAMING_STARTED);
             filter.addAction(Intent.ACTION_DREAMING_STOPPED);
             registerReceiver(mDreamReceiver, filter);
-            checkFullscreen();
         }
```

Comment:
IMHO we should check whether this receiver is actually (still) needed. I'd expect `onResume()` being called when coming back from daydream.
@digitaldan Do you remember details of why this is/was needed?